lib/index.js

```diff
@@ -21,6 +22,8 @@ module.exports = function (ops, coverage) {
   ops = ops || {};
   // Setup
   var ist = ops.istanbul;
+  var outstream = ops.outstream;
+  delete ops.outstream;
```

Comment:
Check out line 31. I'd prefer to add `outstream` to that list rather than delete it explicitly here.